.circleci/config.yml

```
@@ -107,10 +142,20 @@ canadian_rpi: &canadian_rpi
   HOST_ARCH: arm
   BUILD_OS: rpi
 
-linux: &linux
-  BUILD_OS: linux
+canadian_rpi_mac: &canadian_rpi_mac
+  HOST_OS: mac
+  HOST_ARCH: arm
+  BUILD_OS: rpi
+
+mac_xcode: &mac_xcode
+  xcode: 11.7.0
+
+mac_env: &mac_env
+  BUILD_OS: mac
```

I don't think `BUILD_OS` needs to be set, but if it is to `Darwin`.

```suggestion
  BUILD_OS: Darwin
```

I was looking through the `build.sh` script and now I'm confused as to how this works. It should always be a capital `Darwin`, but there are some lowercase `darwin`s in there. That really looks like a bug, so I'm a little confused.

`HOST_ARCH` should be `x86_64`.

Again, I really don't think any of this needs to be defined. In fact, I don't like that `BUILD_OS: linux` exists for the `linux_env`. 

Also, all of the `canadian_rpi_mac` stuff can be removed. The Canadian RPi build only needs to happen once and since it works on Linux, that's good enough.

---

Okay, I'll keep those notes about host arch in mind. As well as removing the canadian rpi (which makes sense)